tests/block_tools.py

```diff
@@ -249,6 +249,7 @@ async def setup_plots(self):
 
         _, loaded_plots, _, _ = load_plots({}, {}, self.farmer_pubkeys, self.pool_pubkeys, None, False, self.root_path)
         self.plots: Dict[Path, PlotInfo] = loaded_plots
+        self._config = load_config(self.root_path, "config.yaml")
```

Comment:
we init self_config in the init()  method why do we need it here as well ?
the config loading in this file is kind of hacky as it is

Reply:
Agreed that it is somewhat messy. This was added to address an issue where create_plots would update the plot_directories value in the config, but our previously read self._config would have the prior value (an empty list in my usage). I've updated the line in question to be more targeted in just updating harvester["plot_directories"] along with a comment.